src/devtools/client/debugger/src/components/Editor/LineNumberTooltip.tsx

```diff
@@ -27,7 +36,10 @@ function getTextAndWarning(analysisPoints?: AnalysisPayload, analysisPointsCount
   if (analysisPoints?.error) {
     return {
       showWarning: false,
-      text: analysisPoints.error === AnalysisError.TooManyPoints ? "10k+ hits" : "Error",
+      text:
+        (analysisPoints.error as AnalysisError) === AnalysisError.TooManyPointsToFind
```

Comment:
For some reason, typescript thinks this things type is `AnalysisError.TooManyPointsToRun | AnalysisError.Unknown`, and so it thinks there is no overlap here. I'm sure it's doing some type inference somewhere, but it's unclear where?

Reply:
Seems a little bit risky that we have hard-coded strings like "10k+ hits" on the frontend. Maybe this number is unlikely to ever change on the backend?